src/pages/Profile/Modal/SubmitModal.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState, useEffect, useContext } from 'react';
+import { ActionText, AttentionTextModal, ColumnContainer, InputsTitleModal, RowContainer, TitleModal } from '../../../globalStyles';
+import { useHttp } from '../../../hooks/http.hook';
+import { Container, Content, InputContainer, InputField, Field, TextArea } from './styled'
+
+export default function SubmitModal({setSubmitModal, queues, setQueues}) {
+    const { request } = useHttp();
+
+    async function handleResetMembers() {
+        const queue = queues.find((item) => item.wrap === true)
+        const data = await request(`${process.env.REACT_APP_API_URL}/api/profile/resetmembers`, 'POST', {id: queue._id});
+        console.log(data);
+
+        queues.forEach(element => {
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/OrangeMGE-ITP/SmartQ-ITP-Front/pullRequest?prid=8748018)
```suggestion
        queues.forEach((element) => {
```